scripts/knative-install.sh

```diff
@@ -12,7 +12,7 @@ kubectl apply --filename https://github.com/knative/serving/releases/download/${
 kubectl apply --filename https://github.com/knative/serving/releases/download/${KNATIVE_VERSION}/serving-core.yaml
 kubectl apply --filename https://github.com/knative/net-istio/releases/download/${KNATIVE_VERSION}/release.yaml
 
-kubectl wait --for=condition=available --timeout=600s deployment/controller -n knative-serving
+kubectl wait --for=condition=available --timeout=600s deployment/controller deployment/webhook -n knative-serving
```

Comment:
Couldn't you change it so that you wait for both ?

Reply:
for both what?

Reply:
nevermind, my eyes are playing tricks on me :smile: